pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/StatisticsConfigurationDao.java

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.ddp.db.dao;
+
+import java.util.List;
+
+import org.broadinstitute.ddp.model.statistics.StatisticsConfiguration;
+import org.jdbi.v3.sqlobject.SqlObject;
+import org.jdbi.v3.sqlobject.config.RegisterConstructorMapper;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.GetGeneratedKeys;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+public interface StatisticsConfigurationDao extends SqlObject {
+
+    @GetGeneratedKeys
+    @SqlUpdate("insert into statistics_configuration (umbrella_study_id, statistics_configuration_type, question_stable_id,"
+            + "answer_value) values (:studyId, :typeId, :questionStableId, :answerValue)")
+    long insertConfiguration(
+            @Bind("studyId") long studyId,
+            @Bind("typeId") long typeId,
```

Comment:
I recommend taking in the `enum` as argument and doing subquery in the sql for the type id.

Reply:
Taken